docs/developer-docs/latest/development/plugins-development.md

```diff
@@ -6,23 +6,28 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/plugins-d
 
 # Plugins development
 
-Strapi allows you to create local plugins that work exactly like the external ones installed with npm or through the Marketplace.
+Strapi allows the development of local plugins that work exactly like the external plugins available from the [Marketplace](). 
 
 :::strapi Extending plugins
 If you would rather extend an existing plugin than create a new one, see the [Plugins extension](/developer-docs/latest/development/plugins-extension.md) documentation.
 :::
 
-## Creating a plugin
+## Create a plugin
```

Comment:
Any reason we are dropping the continuous infinitive versions for the titles? 🤔  
—ing forms have been used (almost) everywhere else in the documentation.

Reply:
It is an accessibility issue. The plain form is easier to translate. https://developers.google.com/style/headings

Reply:
Interesting, thanks for sharing, Shaun!